Polyhedron/demo/Polyhedron/Plugins/Point_set/Point_set_wlop_plugin.cpp

```diff
@@ -98,17 +98,19 @@ void Polyhedron_demo_point_set_wlop_plugin::on_actionSimplifyAndRegularize_trigg
 	      << dialog.neighborhoodRadius() <<" * average spacing)...\n";
 
     // Computes average spacing
-    double average_spacing = CGAL::compute_average_spacing<Concurrency_tag>(points->begin(), points->end(),
+    Point_set::iterator points_begin = (points->nb_selected_points() == 0
+                                        ? points->begin() : points->first_selected());
```

Comment:
You could create a function returning the requested iterator in the item instead of duplicating that code in all plugins.